sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.10.0b2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.10.0 (Unreleased)
```

Comment:
Should we leave a breaking change note (while clarifying that it's not a breaking change between stable versions) that ``validate_authority` support is not available in 1.10.0`, like we did for 1.9.0?